### DISCUSSION TITLE
Lower top and bottom padding of menu items?

### DISCUSSION THREAD

**Discussion by natural-harmonia-gropius** [tomasklaen/uosc#754] [General]
![image](https://github.com/tomasklaen/uosc/assets/50797982/cdc99a8e-3df6-48e8-9e0a-d869d03c49e6)
![image](https://github.com/tomasklaen/uosc/assets/50797982/fb915b44-c69f-4b91-bcbb-4b4fdec689b9)

**Comment by tomasklaen** [comment]
Menu item font size is calculated as a fraction of `menu_item_height`, but if you have a font that has naturally high ratio between line height and glyph height, you can use `font_scale` to fix that.

This seems to be the case for you, since the intended look with the font I'm designing with looks like this (more even horizontal vs vertical padding):

![image](https://github.com/tomasklaen/uosc/assets/47283320/759ac686-6a4d-4006-a022-dd992b557f7a)

So in practice: lower `menu_item_height` and/or bump `font_scale`.
